index.bs

```diff
@@ -3093,6 +3093,42 @@ This [=registration extension=] and [=authentication extension=] enables use of
                     01           -- Subitem 3: CBOR short for Matcher Protection Type Software
     </pre>
 
+## Authenticator Trust Anchor Extension (trustAnchor) ## {#sctn-authenticator-trust-anchor-extension}
+
+This [=registration extension=] allows a [=[RP]=] to specify the acceptable trust anchors (attestation formats)
+to be used when creating the credential. 
+
+: Extension identifier
+:: `trustAnchor`
+
+: Client extension input
+:: Acceptable Trust Anchors:
+
+    <pre class="idl">
+    dictionary biometricCriteria{
+        boolean packed;
+        boolean tpm;
+        boolean androidKey;
+        boolean androidSafetyNet;
+        boolean u2f;
+        };
+    </pre>
+
+: Client extension processing
+:: This extension can only be used during {{CredentialsContainer/create()}}. If the client supports the Authenticator Trust Anchor 
+    Extension, it MUST use the first available  authenticator with an attestation that corresponds to the acceptable trust anchors as specified by the [=[RP]=].
```

Comment:
by "authnr with an attestation" here do you mean to say "authnr utilizing the attestation statement format" ?